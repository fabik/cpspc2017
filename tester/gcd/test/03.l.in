100
340 282 120 113 479 443 215 227 79 201 450 256 382 62 401 352 465 313 215 493 142 381 270 153 384 266 155 466 300 124 169 264 95 8 230 62 50 85 353 392 475 239 99 211 110 395 328 152 86 130 325 364 204 172 454 54 70 333 170 76 352 369 14 101 44 146 297 490 144 23 235 371 416 14 410 440 254 419 214 244 164 171 262 403 107 25 22 79 375 387 119 245 363 225 115 134 448 388 119 28
