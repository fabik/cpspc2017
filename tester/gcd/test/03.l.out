1
494
984
1463
1938
2404
2869
3323
3773
4221
4664
5104
5523
5939
6349
6752
7153
7548
7940
8328
8715
9099
9481
9862
10237
10608
10977
11341
11704
12057
12409
12761
13101
13434
13762
14087
14400
14700
14997
15279
15549
15815
16079
16341
16597
16851
17096
17340
17579
17814
18044
18271
18496
18711
18926
19140
19351
19555
19756
19928
20099
20269
20438
20602
20757
20910
21062
21208
21352
21494
21628
21758
21882
22002
22121
22240
22355
22468
22578
22685
22786
22885
22980
23066
23151
23230
23309
23385
23455
23517
23579
23633
23683
23727
23755
23780
23804
23826
23846
23860
